cmake/analysis/RunCppLint.cmake

```diff
@@ -0,0 +1,24 @@
+include(Macros)
```

Comment:
we aren't using cpp lint right? just clang-tidy

cpplint looked deprecated when i last checked